library/api/src/main/java/com/epmedu/animeal/api/feeding/FeedingPointApiImpl.kt

```diff
@@ -0,0 +1,32 @@
+package com.epmedu.animeal.api.feeding
+
+import CancelFeedingMutation
```

Comment:
There are no such classes in this package. I suppose it is generated data, can you explain how to generate them please?

Reply:
You should either build the project or run `generateApolloClasses` gradle task